integration_tests/integration/confirmArrival.spec.ts

```diff
@@ -15,28 +15,89 @@ context('Confirm Arrival', () => {
     cy.task('stubMissingPrisonerImage')
   })
 
-  it('Should contain a full set of correctly formatted move data', () => {
```

Comment:
Update to tests and differentiate between the PER derived data and that from existing matches